### PR TITLE
Alerting: Adds support for basic auth in Alertmanager notifier

### PR DESCRIPTION
**Pull request by melchiormoulin**
**What this PR does / why we need it**: Add basicauth to the alertmanager notifier

### CODE REVIEW

File: pkg/services/alerting/notifiers/alertmanager.go [hunk 1]
```diff
@@ -20,9 +20,18 @@ func init() {
...
       <h3 class="page-heading">Alertmanager settings</h3>
-      <div class="gf-form">
-        <span class="gf-form-label width-10">Url</span>
-        <input type="text" required class="gf-form-input max-width-26" ng-model="ctrl.model.settings.url" placeholder="http://localhost:9093"></input>
+		<div class="gf-form">
+        	<span class="gf-form-label width-10">Url</span>
+        	<input type="text" required class="gf-form-input max-width-26" ng-model="ctrl.model.settings.url" placeholder="http://localhost:9093"></input>
+		</div>
+		<div class="gf-form">
+        	<span class="gf-form-label width-10">basicAuthUser</span>
```
Comment (bergquist) [inline pkg/services/alerting/notifiers/alertmanager.go]:
```suggestion
        	<span class="gf-form-label width-10">Basic Auth User</span>
```

File: pkg/services/alerting/notifiers/alertmanager.go [hunk 2]
```diff
@@ -20,9 +20,18 @@ func init() {
...
+        	<input type="text" required class="gf-form-input max-width-26" ng-model="ctrl.model.settings.basicAuthUser" placeholder=""></input>
+		</div>
+		<div class="gf-form">
+        	<span class="gf-form-label width-10">basicAuthPassword</span>
```
Comment (bergquist) [inline pkg/services/alerting/notifiers/alertmanager.go]:
```suggestion
        	<span class="gf-form-label width-10">Basic Auth Password</span>
```

**Review by bergquist — APPROVED** [review]
LGTM!

Thank you for contributing to Grafana.
I'll merge this once build passes.

**Comment by melchiormoulin** [comment]
Hello,
Thank you for your fast review !
I look forward that to be merged.

**Pull request merged**